point/src/tpoint_spatialfuncs.c

```diff
@@ -3857,43 +3857,65 @@ tpointseq_linear_at_geometry(const TSequence *seq, Datum geom, int *count)
    * temporal points */
   int countsimple;
   TSequence **simpleseqs = tpointseq_make_simple1(seq, &countsimple);
-  /* Allocate memory for the result */
-  Period ***periods = palloc(sizeof(Period *) * countsimple);
-  int *countpers = palloc0(sizeof(int) * countsimple);
+  Period **allperiods;
```

Comment:
What is the relationship with docbook and C files?

Reply:
None ! Totally independent.

Reply:
Then why are those files changed on a PR about reorganization of docbook?

Reply:
You are right ! I mixed another improvement I was working on this afternoon !

Reply:
hmm, you will eventually learn to have zillions of branches just to make one PR
for example:
https://github.com/cvvergara/MobilityDB/branches/all
and that not counting the ones I didnt push that are in my computer: ;-)

Reply:
I was hoping there was a relationship, so that it could work like in pgRouting where the results of the queries for the documentation are generated automatically, and both queries and results are added to the documentation.